examples/example3.py

```diff
@@ -15,8 +15,8 @@
     lcd.init(w=hres, h=vres)
     disp_drv = lv.disp_create(hres, vres)
     disp_drv.set_flush_cb(lcd.flush)
-    buf1_1 = bytearray(hres * 10 * lv.color_t.__SIZE__)
-    buf1_2 = bytearray(hres * 10 * lv.color_t.__SIZE__)
+    buf1_1 = bytearray(hres * 10 * lv.COLOR_DEPTH // 8)
+    buf1_2 = bytearray(hres * 10 * lv.COLOR_DEPTH // 8)
```

Comment:
`lv.color_format_get_size(color_format)`

Reply:
for this one I believe there is a function called `lv.color_format_get_bpp`

Reply:
Fixed.
Use XRGB888 color format for this **rk043fn48h** display